zomes/hc_zome_trust_atom/src/trust_atom.rs

```diff
@@ -269,45 +314,28 @@ fn convert_link_to_trust_atom(
       TrustAtom {
         source: link_base_b64.to_string(),
         target: link_target_b64.to_string(),
-        content,
-        value,
         source_entry_hash: link_base_b64,
         target_entry_hash: link_target_b64,
-        attributes: BTreeMap::new(), // TODO
+        content: Some(content),
+        value: Some(value),
+        extra: Some(BTreeMap::new()), // TODO
```

Comment:
We should populate this with extra from dht… And integration tests should check it. 

Maybe later we can make this optional.